release-doc-in-process/all_vf.TTL

```diff
@@ -30,22 +30,6 @@ vf:AgentRelationshipRole  a  owl:Class ;
         rdfs:subClassOf   rdf:Property ;
         rdfs:comment   "A relationship role defining the kind of association one agent can have with another." .
 
-vf:ResourceClassification  a    owl:Class ;
-        rdfs:label           "vf:ResourceClassification" ;
-        rdfs:comment   "A resource category, model, taxonomy item, tag, facet value, or other kind of classification." .
-
-vf:ProcessClassification  a           owl:Class ;
-        rdfs:label           "vf:ProcessClassification" ;
-        rdfs:comment   "A process category, model, taxonomy item, tag, facet value, or other kind of classification." .
-
-vf:ExchangeClassification  a           owl:Class ;
-        rdfs:label           "vf:ExchangeClassification" ;
-        rdfs:comment   "An exchange category, model, taxonomy item, tag, facet value, or other kind of classification." .
-
-vf:TransferClassification  a     owl:Class ;
-        rdfs:label               "vf:TransferClassification" ;
-        rdfs:comment   "A transfer category, model, taxonomy item, tag, facet value, or other kind of classification." .
-
```

Comment:
Can we come up with new names and keep these around to build off of for the knowledge layer?  And also the reference properties with new names?